xml/System.Net.Http/HttpClientHandler.xml

```diff
@@ -301,8 +301,8 @@ After NuGet package v4.3.2, the default value of <xref:System.Net.DecompressionM
         <ReturnType>System.Net.ICredentials</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
+        <summary>Gets or sets the credentials to submit to the proxy server for authentication.</summary>
```

Comment:
This property is only used when the "default system proxy" is being used.  This happens only when 'UseProxy=true" and "Proxy=null" (UseProxy and Proxy are HttpClientHandler properties). So, this text needs to be revised.

The property itself is called "DefaultProxyCredentials" because it is the credentials used when the "Default Proxy" is being used.